### DISCUSSION TITLE
Python binding

### DISCUSSION THREAD

**Discussion by Dysta** [tokio-rs/tokio#5848] [General] [answered]
Since Tokio rs provide a low level api for asynchronous with amazing performances, it should be cool to have a python binding for this lib.
I found this one but unfortunately its now in read only https://github.com/PyO3/tokio.

Do you know/plan to have a python binding ?

Ty :)

**Comment by Darksonn** [comment, marked as answer]
The Tokio project has no plans for this. There may be other people who are working on it, but I'm not familiar with that.

**Comment by Dysta** [comment]
Ty for quick answer, have a nice day :)
